### ISSUE TITLE
Http request freezes the update loop

### ISSUE THREAD

**Issue by DeaSTL** [ebernerd/Cobalt#8]
after sending a http within the cobalt.textinput(t) it completely stops the update loop.

https://pastebin.com/NFWr12pC here is the code

**Comment by ebernerd** [comment]
I will look into this issue in the coming hours. In the meantime, I suggest looking at Cobalt-UI, it could clear a lot of your menu overhead.

**Comment by ebernerd** [comment]
Upon further inspection and playing with your code, I don't believe it's actually stopping anything. Your code doesn't do anything once it's received a response. 

First test, I added in an error to print out what your API is returning when logging in (I created a test user), here's what I got:

Upon incorrect username: 
![image](https://user-images.githubusercontent.com/8388562/50133310-1047da00-0259-11e9-98ff-68b35ad12b02.png)
Upon incorrect password: 
![image](https://user-images.githubusercontent.com/8388562/50133300-03c38180-0259-11e9-968e-c77215854837.png)
Upon correct username and password: ![image](https://user-images.githubusercontent.com/8388562/50133281-e7274980-0258-11e9-87ce-c8e7794be893.png)

Second test, I had your program return to the main menu once a response was given, and the keyboard could still be used, just as before: ![image](https://i.imgur.com/Hn7fmJv.gif)

Like I said before, Cobalt-UI exists and should do what you want in terms of menus. If you must stick this way, here are my tips:
- Refrain from using sleep statements, they're not necessary and stop everything.
- Use local variables - you might get unexpected behavior if you close and then reopen the same program if you're using global variables

If you still have issues, let me know.
